### DISCUSSION TITLE
[Arcropolis 3.9.0] Game crashes using mods on a specific slot and slot addition

### DISCUSSION THREAD

**Discussion by Asunemara** [Raytwo/ARCropolis#449] [Issues]
System Firmware Version:
15.0.1

Atmosphere Version:
1.4.0

Smash Version:
13.0.1

ARCropolis Version:
3.9.0 (It happens since 3.7.0)

Can the crash be consistently reproduced?
Yes

At what moment does the crash happen?
When you select a specific character with added slots the game crashes.

Details:
Since ARCropolis 3.7.0 when I put a mod on a specific slot from a specific character and I select any slot of that character the game crashes at the vs. screen, but if I leave that slot without mods everything works fine.

I found this to happen with Bowser on c02, Banjo & Kazooie on c01 and Palutena on c05 but I'm sure that this can happen with other characters too.

Plugins turned on:
libarcropolis.nro
libone_slot_eff.nro (the game crashes without it too)

Crash report:
[01667408610_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/9922887/01667408610_01006a800016e000.log)

Reproducible Modpack:
[mods.zip](https://drive.google.com/file/d/1_jMeJkUMeU0w7tLniV_WUds6Ivpavk4l/view?usp=share_link)

**Comment by mRama3415** [comment]
I’ve had this happen on, seems to be, specific slots as well. Also some characters that have additional slots will crash my game at the result screen no matter what additional slot I’m using, while other characters work fine. So far I’ve found little mac and dr mario cause these issues consistently

**Comment by ElaineLMN** [comment]
Wanted to mention that I've been having almost the exact same issue but also on base slots (c00-c07)! Created a discussion for it here: #451

**Comment by secretchaos1** [comment]
I've seen a similar issue to this sort of thing but had no problems with Banjo slots. Bayonetta on the other hand would crash with any c00 based slot (including the untouched vanilla c00 slot) when I had a mod on c04, moving it to a different slot resolved the issue. I've been testing alphabetically and noticed Bowser was having similar issues with crashing + the vanilla slot having a missing eye texture but I haven't narrowed down the problem slot yet. I remember back when file unsharing was being researched there was a page that documented which slots were the "source" slots for each character, I wonder if there's any correlation here and that's possibly why my vanilla slots aren't working?
Either way, skipping over the affected slots seems to be my workaround for now, just takes some trial and error to find them.
